packages/cli/test/unit/commands/git/disconnect.test.ts

```diff
@@ -12,6 +12,51 @@ describe('git disconnect', () => {
   const fixture = (name: string) =>
     join(__dirname, '../../../fixtures/unit/commands/git/connect', name);
 
+  it('tracks subcommand invocation', async () => {
```

Comment:
mostly copypasta from a similar test. We'll swing through cleaning these up some day.